lib/experimental/Widgets/Content/Highlights/WidgetHighlightButton/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Icon, IconType } from "@/components/Utilities/Icon"
+import { cn } from "@/lib/utils"
+
+type Props = {
+  label: string
+  icon: IconType
+  iconClassName?: string
+  count: number
+  onClick?: () => void
+}
+
+export type WidgetHighlightButtonProps = Props
+
+type WrapperProps = {
+  onClick?: (ev: React.MouseEvent<HTMLAnchorElement>) => void
+  children: React.ReactNode
+}
+
+const Wrapper: React.FC<WrapperProps> = ({ onClick, children }) => {
+  const className =
+    "block rounded-lg border border-solid border-transparent p-[1px]"
+
+  return onClick ? (
+    <a
```

Comment:
What's the purpose of having an `a` tag without an `href`? It won't be read as a link by assistive technologies, and it won't be focusable by default, so we need to add a `tabIndex`, and we just use it for an `onClick`, why not a `button` directly? 🤔